cmd/util/ledger/migrations/cadence_values_migration_test.go

```diff
@@ -830,6 +829,8 @@ func TestProgramParsingError(t *testing.T) {
 		},
 	)
 
+	cadenceValueMigratorReporter := rwf.reportWriters["cadence-value-migrator"]
```

Comment:
Maybe introduce a constant for the migration name and use it in its definition and here